docs/howto/create-a-hollow-jar/index.adoc

```diff
@@ -31,8 +32,7 @@ $ mvn package
 
 . Execute the hollow JAR:
 +
-Execute the resulting `-hollowswarm.jar` file using the Java binary and pass
-the `.war` file with your application as the first argument:
+Execute the resulting `-hollowswarm.jar` file using the Java binary and pass the `.war` file with your application as the first argument:
```

Comment:
I think Ken also has a PR for this, but it's apparently `-hollow-swarm.jar` and not `-hollowswarm.jar`

Reply:
I couldn't find the PR, but in the [existing WFS guide](https://wildfly-swarm.gitbooks.io/wildfly-swarm-users-guide/content/getting-started/tooling/maven-plugin.html), it's really `-hollow-swarm.jar`, so I'll change it here as well.